components/collapsible-panel/collapsible-panel.js

```diff
@@ -530,7 +542,6 @@ class CollapsiblePanel extends SkeletonMixin(FocusMixin(RtlMixin(LitElement))) {
 	_toggleExpand() {
 		if (this.skeleton) return;
 		this.expanded = !this.expanded;
-		this.focus();
```

Comment:
I'm wondering... do we even need this `focus()` call at all? It's called from `_handleHeaderClick` (the `<div>` around the whole header and the button around the heading) and `_handlePanelClick` (the `<div>` around the _entire_ thing, which seems to only be there to expand when the summary is clicked). In all those cases, moving focus to the button doesn't seem necessary to me.

Reply:
Without it it seems focus goes to the `body` because the `click`s are stopped short of (re)triggering focus.

Reply:
Yeah maybe, and I agree with your other comment. Maybe we could try just keeping the very top-level click handler? Might need a special one on the button to handle keyboard, but the others seem unnecessary given that they should bubble up anyway.

Reply:
Thanks for the input on this. I'll keep this click handler idea and work on restructuring the click handlers.